promotion/controllers_test.go

```diff
@@ -899,7 +899,7 @@ func (suite *ControllersTestSuite) TestClaimCompatability() {
 			PublicKey:   hexPublicKey,
 			LastBalance: nil,
 		}
-		suite.Require().NoError(pg.InsertWallet(w), "could not insert wallet")
+		suite.Require().NoError(pg.UpsertWallet(w), "could not insert wallet")
```

Comment:
Should this message be `could not upsert wallet` ?

Reply:
the upsert is just inserting since our dbs will be clean at the start of the test